msg/Defect.msg

```diff
@@ -1,10 +1,8 @@
 # DISCLAIMER: this will not be the final messagge to describe a defect.
 # This is a temporary placeholder for demonstration purposes.
 
-# Severity levels
-# 1 = class A
-# 2 = class B
-# 3 = class C
+uint8 number
```

Comment:
Why did you add the `number` here?
I don't think we shall hard-code an ID of the defect.
Reading the updated docs, you state that it is the order for the defect to be reworked, but I think that this concept is misleading.
After the you solve the orienteering, you provide a list of **ordered** defects, i.e., the plan that the robot shall follow.

Reply:
Yes, i think it is better to have one way to index nodes,  for me i think an ID would be helpful. Do you have any other suggestions?

Reply:
> Why did you add the `number` here? I don't think we shall hard-code an ID of the defect. Reading the updated docs, you state that it is the order for the defect to be reworked, but I think that this concept is misleading. After the you solve the orienteering, you provide a list of **ordered** defects, i.e., the plan that the robot shall follow.

Yes, i think it is better to have one way to index nodes,  for me i think an ID would be helpful. Do you have any other suggestions?

Reply:
In my opinion, the `number` index you are going to provide here will match exactly the index within `magician_msgs/Defect[] roadmap` vector that you will provide as output of the orienteering problem, so its a repetition.
Moreover, the "Defect" message could potentially be shared with other topics/services that might not strictly require an *indexing feature*.

If you really think that your message requires an ID, maybe create a new message (e.g. `OrienteeringDefect`) within a subfolder that will contains `magician_msgs/Defect data`, and everything else you need for the orienteering problem (that other services might not need)